src/mobility/components/PricingPlan.tsx

```diff
@@ -69,6 +82,11 @@ const PriceInfo = ({price, pricingSegment, unit}: PriceInfoProps) => {
     <VehicleStat
       primaryStat={`${formatPrice(pricingSegment, language)} kr/${unit}`}
       secondaryStat={t(ScooterTexts.pricingPlan.price(price))}
+      secondaryStatStyle={
+        price > 0 && eligibleBenefits?.includes('free-unlock')
+          ? {textDecorationLine: 'line-through'}
+          : undefined
+      }
```

Comment:
My slight preference is to encapsulate the lower level styling into the component, and rather use a variable to switch between styles. Would be something like `isFree` in this case. I feel it is easier to enforce the design system in the code that way.